docs/hardware/README.md

```diff
@@ -0,0 +1 @@
+# Hardware Guide
```

Comment:
@123MwanjeMike please advice on the content for this file

Reply:
Hi @HassanBahati.
We can add there a generic brief summary about the hardware guide